configure.ac

```diff
@@ -338,14 +345,18 @@ fi
 AM_CONDITIONAL([ENABLE_LASTLOG], [test "x$enable_lastlog" != "xno"])
 
 AC_SUBST([LIBSYSTEMD])
-if test "X$enable_logind" = "Xyes"; then
+if test "X$enable_logind" != "Xno"; then
 	AC_CHECK_LIB([systemd], [sd_session_get_remote_host],
-		[enable_logind="yes"; [LIBSYSTEMD=-lsystemd];
+		[enable_logind="yes"
+		LIBSYSTEMD=-lsystemd
```

Comment:
What's the rationale for this?

Reply:
To fix overcomplicated and confusing code.
Second level of quoting specifically around `LIBSYSTEMD=-lsystemd` is not needed, confusing and inconsistent.
When second level of quoting is used, the selected part treated literally, but nothing needs to be treated literally in this part (prevent expansion).
Also, when each assignment has its own line, the shell code is better (in terms of readability).

Reply:
Please separate those cosmetic changes from the main commit.

Reply:
Converted to four commits.